source/Playnite/Settings/PlayniteSettings.cs

```diff
@@ -1907,6 +1907,54 @@ public bool ShowElevatedRightsWarning
             }
         }
 
+        private string dateTimeFormatAdded = Constants.DefaultDateTimeFormat;

```

Comment:
Is there a reason why there's no option for "Last Played" field?

Reply:
Because LastPlayed is already using a converter that displays dates or a string

![image](https://user-images.githubusercontent.com/1389286/191069385-b1570711-525d-4322-ae3f-f8502326172c.png)

![image](https://user-images.githubusercontent.com/1389286/191069718-0a695a10-a093-4e33-9b24-b29009dd8d52.png)


In theory it could be added but it could be a little difficult because it would require a solution different from the other ones as it would need to be configurable to use the current behavior or display only dateTime strings. Maybe for a future update?

Reply:
Done as part of https://github.com/JosefNemec/Playnite/pull/3024/commits/8a67067af4f422b61c668fc1b8bf068cccfae211